source/import-export.txt

```diff
@@ -158,6 +162,46 @@ Procedure
       .. include:: /includes/steps/export-filter.rst
 
       .. include:: /includes/fact-export-progress-bar.rst
+   
+   .. tab:: Export Aggregation Results
+      :tabid: export-Aggregation
+
+      To export results from your aggregation pipeline:
+
+      .. procedure::
+         :style: normal
+
+         .. step:: Connect to the deployment containing the :doc:`collection </collections>` you wish to export data from.
```

Comment:
Actually, this still isn't showing as bold. Maybe it doesn't like the `:doc:` role being used in the step title? Can you try removing that and seeing if that puts the step in bold?

Reply:
Oh you're right, maybe I just needed to refresh. Sorry about that